scripts/install.sh

```diff
@@ -93,6 +93,21 @@ configure_systemd() {
         status "Adding ollama user to video group..."
         $SUDO usermod -a -G video ollama
     fi
+    status "Creating ollama systemd service directory if not exists..."
+    $SUDO mkdir -p /etc/systemd/system/ollama.service.d
+
+    status "Creating environment.conf file if not exists..."
+    ENV_CONF_FILE="/etc/systemd/system/ollama.service.d/environment.conf"
+    if [ ! -f "$ENV_CONF_FILE" ]; then
+        cat <<EOF | $SUDO tee "$ENV_CONF_FILE" >/dev/null
+[Service]
+# Conditionally include EnvironmentFile directive
+EnvironmentFile=/etc/default/ollama
+EOF
+    fi
+
+    status "Touching /etc/default/ollama file to create it if not exists..."
+    $SUDO touch /etc/default/ollama
```

Comment:
This should check if the file exists and create it with some placeholders if it does not

```sh
if [ ! -f '/etc/default/ollama' ]; then
  cat <<EOF | $SUDO tee /etc/default/ollama >/dev/null
# OLLAMA_HOST sets the address and/or port Ollama binds to
# OLLAMA_HOST="127.0.0.1:11434"
#
# OLLAMA_ORIGINS sets additional web origins Ollama accepts
# OLLAMA_ORIGINS=""
#
# OLLAMA_MODELS sets the location where models are stored
# OLLAMA_MODELS="/usr/share/ollama/.ollama/models
EOF
fi
```

Reply:
I like the addition, my intent was just to make it so people would have the old settings after update.  this is better thanks.